modules/net-vpc-firewall/README.md

```diff
@@ -138,10 +138,10 @@ module "firewall" {
   project_id       = "my-project"
   network          = "my-network"
   factories_config = {
-
+    rules_folder    = "config/firewall"
+    cidr_tpl_fil    = "config/cidr_template.yaml"
```

Comment:
Thank you for spotting that and contribution. 
A small correction, suppose `e` is missing, should be `cidr_tpl_file`.